src/BitzArt.XDoc/Utility/XmlMemberNameResolver.cs

```diff
@@ -91,22 +77,22 @@ private static void EnsureContainsTypeSeparator(string xmlDocumentationMemberNam
     /// <returns>
     /// A read-only collection of parameter type names as strings. Returns an empty collection if no parameters are found.
     /// </returns>
-    public static IReadOnlyCollection<string> ResolveMethodParameters(string xmlDocumentationMemberName)
+    public static IReadOnlyCollection<string>? ResolveMethodParameters(string xmlDocumentationMemberName)
     {
         var parameterListStartIndex = xmlDocumentationMemberName.IndexOf('(');
 
         if (parameterListStartIndex == -1)
         {
-            // No parameters found
-            return [];
+            // No parameter list found
+            return null;
```

Comment:
I would suggest not to change this code, the current implementation return an empty list, and it preventing unexpected NullReferenceException and allow avoid extra null checks.